share/jupyter/kernels/xr/resources/execute.R

```diff
@@ -168,4 +160,17 @@ execute <- function(code, execution_counter, silent = FALSE) {
     tryCatch(send_plot(last_plot), error = handle_error)
   }
 
+  if (isTRUE(last_visible)) {
+    obj <- .Last.value
+
+    mimetypes <- if (getOption('jupyter.rich_display')) {
```

Comment:
perhaps this should only be `text/plain`. I need to look how things are done e.g in 🐍